setup.cfg

```diff
@@ -51,7 +51,7 @@ install_requires =
     eralchemy==1.2.*
     hieroglyph>=2.1.0
     setuptools>=50
-    sphinx>=4.4
+    sphinx>=4.4,!=7.2.*
```

Comment:
Are we expecting this issue to be fixed in 7.3?

Reply:
Don't know when it will be fixed, but hopefully? I've opened an issue: https://github.com/sphinx-doc/sphinx/issues/11631